lib/manageiq/session.rb

```diff
@@ -44,7 +44,8 @@ def self.store=(session_store)
     # Configures the session store to be used by the Rails application
     # (Vmdb::Application) and logs what is being used.
     def self.configure_session_store(adapter)
-      Vmdb::Application.config.session_store(*adapter.session_store_config_args)
+      args = adapter.session_store_config_args
+      Vmdb::Application.config.session_store(args.first, **args.second)
```

Comment:
I was going to say:

> Since we own what we are modifying here, can we just adjust the `session_store_config_args`...

But I think the problem here is that we can't do that from that side because we are still going to have to pass back an array.

That said, maybe just calling the underlying methods instead of `.first` and `.second` would be more obvious:

```diff
-       Vmdb::Application.config.session_store(*adapter.session_store_config_args)
+       Vmdb::Application.config.session_store(adapter.type, **adapter.session_options)
```

Reply:
I like this approach ☝️ I think it adds some clarity as well as that `session_store_config_args` method isn't really buying us anything.